dbt_project.yml

```diff
@@ -1,10 +1,10 @@
 name: 'readmissions'
-version: '0.1.1'
+version: '1.0.0'
```

Comment:
I have tested with this version when doing the Freebird data profiling in both Snowflake and Redshift.